src/hm/hmRadio.h

```diff
@@ -285,16 +311,18 @@ class HmRadio : public Radio {
                                 ah::dumpBuf(p.packet, p.len, 1, 4);
```

Comment:
Sollten wir hier nicht erst prüfen ob es ein TX Command (MainCmd bzw `p.packet[0] < 0x80`) oder ein RX Command (MainCmd bzw. `p.packet[0] > 0x80`) ist ?
Das wäre auch für einen Promiscuous mode sinnvoll.

Danach können wir immer noch die Inverter Serial und die Paket Nummer prüfen, aber das gehört m.E. dann bereits zum Paket Zähler, CRC8 Verifikation und anschließenden Payload Analyse.
Für ein möglichst schnelles Umschalten auf Empfang für das nächste RX Paket könnte man das eigentlich auch erst nachgelagert für den kompletten empfangenen Paketstack machen.
Wenn ein Paket fehlt, dann müssen wir eh erst wieder in den TX Modus wechseln.

Reply:
Die Prüfung auf den Inverter sollte in jedem Fall hier bleiben, die war nämlich früher nur an einer anderen Stelle. Das hat dann häufiger zu kaputten payloads geführt, weil die Daten eben vom falschen Inverter stammen...

Zum Rest sollte vielleich @lumapu was sagen, da stehe ich grade im Wald, was gemeint sein könnte.